src/pages/FindWorkflow/FindWorkflowModal.js

```diff
@@ -22,11 +27,11 @@ const styles = {
 
 const suggestedWorkflowsList = [
   {
-    method_name: 'mock_method_1',
-    method_description: 'mock_method_1 description. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+    method_name: 'Optimus',
+    method_description: 'The optimus 3 pipeline processes 10x genomics sequencing data based on the v2 chemistry. It corrects cell barcodes and UMIs, aligns reads, marks duplicates, and returns data as alignments in BAM format and as counts in sparse matrix exchange format.',
     method_source: 'GitHub',
```

Comment:
I think the source for this is [Dockstore](https://dockstore.org/workflows/github.com/broadinstitute/warp/Optimus:Optimus_v5.5.0?tab=info) based on the workflow config page in Terra

Reply:
This means that we won't be able to import this method right? Is there a check in CBAS that stops non-Github workflows from being imported?

Reply:
Ah you are right. We won't be able to import if the source is Dockstore. I guess then we can leave it as Github for now and change it when we add Dockstore integration.

Reply:
Right now this "source" is informational only, so we can put whatever value we want there

Reply:
We can check with Sara when she's back